vulnerabilities/api.py

```diff
@@ -175,9 +203,9 @@ def filter_purl(self, queryset, name, value):
 
 
 class PackageViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = Package.objects.all()
+    queryset = Package.objects.all().order_by("id")
     serializer_class = PackageSerializer
-    paginate_by = 50
+    pagination.PageNumberPagination.page_size = 10
```

Comment:
This seems very wrong plus it's unrelated to this PR.

Reply:
I have cut a branch from #830, I will rebase this branch once #830 get's merged